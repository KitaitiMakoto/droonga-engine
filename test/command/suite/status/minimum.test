{
  "type": "status",
  "dataset": "Default",
  "body": {}
}
