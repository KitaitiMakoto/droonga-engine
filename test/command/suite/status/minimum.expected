{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "status.result",
  "body": {
    "nodes": {
      "127.0.0.1:23003/droonga": {
        "live": true
      }
    }
  }
}
